### ISSUE TITLE
[A11y] The back to top button's hover style makes it almost disappear

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#60]
Windows 10/Chrome 81 (latest)

Go to https://www.ibm.com/able/

Hover over the "back to top" button.
It disappears. (Unless you look really closely).

![image](https://user-images.githubusercontent.com/3331913/81840133-8ae05c00-9516-11ea-89b2-ce55acc6f461.png)

A disappearing button feels weird. I was just trying to figure out what it was, and I hoped it would have a tooltip.

The button markup is:
```html
<button class="BackToTopBtn-module--button--32mr1" type="button" aria-label="Back to Top">
  ...
</button>
```

And the problem css is:
```css
.BackToTopBtn-module--button--32mr1:hover {
    background: var(--cds-inverse-01);  /* << this color is the same as the page background */
    border-color: transparent;
}
```

**Comment by tombrunet** [comment]
Bug in Gatsby Carbon theme: https://github.com/carbon-design-system/gatsby-theme-carbon/blob/master/packages/gatsby-theme-carbon/src/components/BackToTopBtn/BackToTopBtn.module.scss#L19

I'm talking to their designers about what this should do, so we can override and then we'll contribute it back.

**Comment by tombrunet** [comment]
Discovered on the Gatsby demo site that they went with a black background. Fix will be deployed on Monday.

**Comment by tombrunet** [comment]
Deployed
